README.md

```diff
@@ -53,7 +58,9 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
+O(n/2 + 1)  (which would simplify to O(n)...just in case)
+Since there is symmetry, we can always assume there are two of the same tile
+and then we must also consider the extra one in the center.
```

Comment:
# -1

You're correct, but I'm not sure about your explanation: are you talking about speed or memory? Any talk of memory should include the parser's stack.